### PR TITLE
Update question 191

### LINKED ISSUE #122: Update question 191
### Thanks for helping!

Thank you for helping to port [this question](../blob/master/questions/191/) from C++17 to C++23.

You'll find the source code in [question.cpp](../blob/master/questions/191/question.cpp), this should normally not need modification.

In C++17, the correct answer is:
> The program is guaranteed to output: `01`

Please verify that this is still correct in C++23. If it is, please do the following:
- [ ] Update [explanation.md](../blob/master/questions/191/explanation.md):
  - [ ] Refer to the correct section numbers
  - [ ] Use updated quotes from those sections (the wording might have changed)
  - [ ] Make sure the rest of the text in the explanation is consistent with the new standard
- [ ] Update [hint.md](../blob/master/questions/191/hint.md) if needed (usually not needed)

*Note that some older questions are missing references to the standard, we don't need to fix that now.*

If you get stuck, please describe the problem in detail in a comment to the issue, and a maintainer will add the `help wanted` tag so others can chime in.

If the correct answer has changed from C++17 to C++23, you can either just leave a comment in this issue, or see the instructions for [updating meta data](../blob/master/METADATA_HOWTO.md).

### PR DESCRIPTION
**Pull request by ani-hovhannisyan**
Fixes #122

### CODE REVIEW

File: questions/191/explanation.md [hunk 1]
```diff
@@ -2,5 +2,5 @@ Due to the `extern "C"` specifications, `A::x` and `B::x` actually refer to the
 
 `x` is first initialized to `0`, then `main()` starts, `0` is printed, `x` is incremented to `1`, and finally `1` is printed.
 
-§[dcl.link]¶6 in the C++ standard:
-"Two declarations for a variable with C language linkage with the same name (ignoring the namespace names that qualify it) that appear in different namespace scopes refer to the same variable."
+§[dcl.link]¶7 in the C++23 standard:
```
Comment (tocic) [inline questions/191/explanation.md:5]: Let's use just "C++" here without specifying the exact version. See https://github.com/knatten/cppquiz23/pull/201#discussion_r1658104656.

File: questions/191/explanation.md [hunk 2]
```diff
@@ -2,5 +2,5 @@ Due to the `extern "C"` specifications, `A::x` and `B::x` actually refer to the
...
+"Two declarations declare the same entity if they (re)introduce the same name, ..."
```
Comment (tocic) [inline questions/191/explanation.md:6]: I think you've omitted an importart part of the explanation, namely "_(...) one declares a function or variable with C language linkage, and the other declares such an entity (...)_".

**Review by tocic — CHANGES_REQUESTED** [review]
Please rebase your branch so it doesn't contain changes to another question.

**Comment by ani-hovhannisyan** [comment]
> Please rebase your branch so it doesn't contain changes to another question.

Please remove this PR, sorry for inconvenience, will PR again with rebased version.

**Pull request closed without merging**